OpenSim/Simulation/Model/PhysicalOffsetFrame.h

```diff
@@ -56,7 +56,9 @@ class OSIMSIMULATION_API PhysicalOffsetFrame : public OffsetFrame<PhysicalFrame>
     // CONSTRUCTION
     //--------------------------------------------------------------------------
     /** Constructors are defined by the OffsetFrame base class */
+#ifndef SWIG    
     using OffsetFrame<PhysicalFrame>::OffsetFrame;
+#endif
```

Comment:
This change is to avoid the following warning:

```
/Users/chris/repos/opensim/13/opensim-core/OpenSim/Simulation/Model/PhysicalOffse
tFrame.h:59: Warning 315: Nothing known about 'OpenSim::OffsetFrame< OpenSim::Phy
sicalFrame >::OffsetFrame'.   
```

We already wrap `OpenSim::OffsetFrame< OpenSim::PhysicalFrame >`, so apparently that wasn't sufficient for avoiding this warning. Hence the `ifndef`.

Reply:
OK. Thanks for the check. Likely future releases of SWIG will remedy the issue. We can attempt to remove when that happens.